pkg/crc/preflight/preflight_windows.go

```diff
@@ -63,55 +64,24 @@ var hypervPreflightChecks = []Check{
 			return nil
 		},
 		fixDescription: "Creating crc-users group",
-		fix: func() error {
-			if _, _, err := powershell.ExecuteAsAdmin("create crc-users group", "New-LocalGroup -Name crc-users"); err != nil {
-				return fmt.Errorf("failed to create 'crc-users' group: %v", err)
-			}
-			return nil
-		},
-		cleanupDescription: "Removing 'crc-users' group",
-		cleanup: func() error {
-			if _, _, err := powershell.ExecuteAsAdmin("remove crc-users group", "Remove-LocalGroup -Name crc-users"); err != nil {
-				return fmt.Errorf("failed to remove 'crc-users' group: %v", err)
-			}
-			return nil
-		},
-		labels: labels{Os: Windows},
+		flags:          NoFix | StartUpOnly,
+		labels:         labels{Os: Windows},
 	},
 	{
 		configKeySuffix:  "check-user-in-hyperv-group",
-		checkDescription: "Checking if current user is in Hyper-V group and crc-users group",
-		check: func() error {
-			if err := adminHelperGroup.check(); err != nil {
-				return err
-			}
-			return hypervGroup.check()
-		},
-		fixDescription: "Adding current user to group",
-		fix: func() error {
-			m := crcerrors.MultiError{}
-			if err := adminHelperGroup.check(); err != nil {
-				m.Collect(adminHelperGroup.fix())
-			}
-			if err := hypervGroup.check(); err != nil {
-				m.Collect(hypervGroup.fix())
-			}
-			if len(m.Errors) == 0 {
-				return nil
-			}
-			if m.Errors[0] == errReboot {
-				return errReboot
-			}
-			return m
-		},
-		labels: labels{Os: Windows},
+		checkDescription: "Checking if current user is in Hyper-V",
+		check:            checkIfUserPartOfHyperVAdmins,
```

Comment:
This removes the check for the admin-helper group, and only keep the check for hyper-v, this probably deserves to be done separately for a better explanation.

Reply:
Added a separate commit for this